impl/src/test/java/org/ehcache/docs/GettingStarted.java

```diff
@@ -163,22 +163,38 @@ public void defaultSerializers() throws Exception {
   @Test
   public void byteSizedTieredCache() {
     // tag::byteSizedTieredCache[]
-    CacheConfiguration<Long, String> cacheConfiguration = CacheConfigurationBuilder.newCacheConfigurationBuilder(Long.class, String.class)
+    CacheConfiguration<Long, String> usesConfiguredInCacheConfig = CacheConfigurationBuilder.newCacheConfigurationBuilder(Long.class, String.class)
         .withResourcePools(ResourcePoolsBuilder.newResourcePoolsBuilder()
             .heap(10, MemoryUnit.KB) // <1>
             .offheap(10, MemoryUnit.MB)
             .build())
-        .add(new DefaultSizeOfEngineConfiguration(1000, 1000)) // <2>
```

Comment:
From conversation with @chrisdennis and @AbfrmBlr:
`new DefaultSizeOfEngineConfiguration(long size, MemoryUnit unit, long objectGraphSize)`